### ISSUE TITLE
No corresponding source for the libffi build

### ISSUE THREAD

**Issue by mwcampbell** [NativeScript/ios-runtime#369]
There is no pointer to the source code for the specific version of libffi that is included in binary form in this repository, nor instructions to rebuild it. This means that if I find a problem with libffi, I can't debug it.

**Comment by fealebenpae** [comment]
Hi @mwcampbell,

We haven't included the sources for libffi because we built it from two separate code bases. We built it once for arm64 with this revision https://github.com/atgreen/libffi/commit/536366349ced9d57c873d64e219ba30992a83c06 and we built it for armv7, i386 and x86_64 with this revision https://github.com/atgreen/libffi/commit/20562ac0427c3578250d04c6e34fb0127d4551cf (release 3.2.1). If we could automate the build of libffi we would have included it as a submodule like we did WebKit, but as it is right now we can't build armv7 and arm64 from the same revision.

The source isn't otherwise modified, so combining the builds of those two revisions in a fat binary will yield the same libffi library that's currently in the repo.

Unfortunately, debugging libffi right now means building the correct revision locally and replacing the production library in the repo with the debug build.
